Tests/Integration/MSFT_XIISLogging.config.ps1

```diff
@@ -0,0 +1,27 @@
+configuration MSFT_xIisLogging_Rollover
+{
+    Import-DscResource -ModuleName xWebAdministration
+
+    xIisLogging Logging
+    {
+        LogPath = 'C:\IISLogFiles'
+        Logflags = @('Date','Time','ClientIP','UserName','ServerIP')
+        LogFormat = 'W3C'
```

Comment:
This doesn't exist as a parameter in your get/set/test-targetresource which is what's causing the failure in the integration test.  Seems like it's something you want to exist given the testing for it.